### PR TITLE
Orders document viewer [delivers #157207140]

### PR DESCRIPTION
**Pull request by jim**
## Description

This new screen can be accessed using the path `/moves/:move_id/orders` since it is specific to viewing a move's orders.

## Reviewer Notes

* The orders number is not being returned by the server, but will be once other in progress work is completed.
* PDFs are rendered as a link to the file. Displaying them is not part of this story.

## Code Review Verification Steps

* [ ] All tests pass.
* [ ] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [ ] This works in IE.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/157207140) for this change

## Screenshots

screen shot 2018-05-17 at 10 01 56 am

### CODE REVIEW

File: src/scenes/Office/MoveInfo.jsx [hunk 1]
```diff
@@ -203,7 +203,7 @@ class MoveInfo extends Component {
                       icon={faExclamationCircle}
                     />
                     <a href={upload.url} target="_blank">
```
Comment (kilbergr) [inline src/scenes/Office/MoveInfo.jsx]: I'm pretty sure this should lead to your fancy new viewer, not to the upload url.

File: src/scenes/Office/OrdersInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+import { connect } from 'react-redux';
+import { compact, get } from 'lodash';
+
+import LoadingPlaceholder from 'shared/LoadingPlaceholder';
+import Alert from 'shared/Alert';
+import { loadMoveDependencies } from './ducks.js';
+import { formatDate, formatDateTime } from './helpers';
+
+// import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+// import faExclamationCircle from '@fortawesome/fontawesome-free-solid/faExclamationCircle';
```
Comment (kilbergr) [inline src/scenes/Office/OrdersInfo.jsx]: rm

File: src/scenes/Office/index.jsx [hunk 3]
```diff
@@ -51,6 +52,10 @@ class OfficeWrapper extends Component {
                   component={MoveInfo}
                 />
                 <PrivateRoute path="/queues/:queueType" component={Queues} />
+                <PrivateRoute
+                  path="/moves/:moveId/orders"
+                  component={OrdersInfo}
```
Comment (kilbergr) [inline src/scenes/Office/index.jsx]: Should this be `queues/:queueType/moves/:moveId/orders`? I don't much care, it's just that the basics tab works as `http://officelocal:3000/queues/:queueType/moves/:moveId/basics`.

File: src/scenes/Office/OrdersInfo.jsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+            {uploads}
+          </div>
+          <div className="usa-width-one-third orders-page-fields">
+            <h2 className="usa-heading">{name}</h2>
+
+            <PanelField title="Move Locator">{move.locator}</PanelField>
+            <PanelField title="DoD ID">{serviceMember.edipi}</PanelField>
+
+            <h3>
+              Orders {orders.orders_number} ({formatDate(orders.issue_date)})
```
Comment (kilbergr) [inline src/scenes/Office/OrdersInfo.jsx]: orders_number isn't getting loaded, and also I don't think you need it here (unless I'm looking at an old wireframe).

File: src/scenes/Office/OrdersInfo.jsx [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+            </h3>
+            {uploads.length > 0 && (
+              <p className="uploaded-at">
+                Uploaded{' '}
+                {formatDateTime(orders.uploaded_orders.uploads[0].created_at)}
+              </p>
+            )}
+
+            <PanelSwaggerField
+              fieldName="orders_number"
```
Comment (kilbergr) [inline src/scenes/Office/OrdersInfo.jsx]: orders_number not getting loaded

File: src/scenes/Office/OrdersInfo.jsx [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+            <PanelField title="New Duty Station">
+              {orders.new_duty_station && orders.new_duty_station.name}
+            </PanelField>
+
+            <PanelField
+              className="Todo"
+              title="Dependents"
+              value={orders.has_dependents ? 'Authorized' : 'Not authorized'}
+            />
+            <PanelField className="Todo" title="Dept. Indicator" />
```
Comment (kilbergr) [inline src/scenes/Office/OrdersInfo.jsx]: Hasn't what we need to load this landed?

File: src/scenes/Office/OrdersInfo.jsx [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+            <PanelField className="Todo" title="TAC" />
```
Comment (kilbergr) [inline src/scenes/Office/OrdersInfo.jsx]: Ditto

File: src/scenes/Office/MoveInfo.jsx (hunk 1 above)
Comment (jim) [reply src/scenes/Office/MoveInfo.jsx]: There is a separate story for this: https://www.pivotaltracker.com/story/show/157546611

File: src/scenes/Office/index.jsx (hunk 3 above)
Comment (jim) [reply src/scenes/Office/index.jsx]: I don't think it's specific to any queue, and I can imagine it being opened from any of them. I'm pretty flexible on this though.

File: src/scenes/Office/OrdersInfo.jsx (hunk 4 above)
Comment (jim) [reply src/scenes/Office/OrdersInfo.jsx]: It's not getting returned by the server yet, but I do see it in the wireframe [here](http://slicedbreadclient.com/dds/dp3/wireframes/BackOffice/round4/Orders-2.html):

screen shot 2018-05-18 at 9 05 26 am

File: src/scenes/Office/OrdersInfo.jsx (hunk 5 above)
Comment (jim) [reply src/scenes/Office/OrdersInfo.jsx]: See above. I'll add `.Todo` to it.

File: src/scenes/Office/OrdersInfo.jsx (hunk 6 above)
Comment (jim) [reply src/scenes/Office/OrdersInfo.jsx]: It has, but it hadn't when this was written. I'll remove the class.

File: src/scenes/Office/OrdersInfo.jsx (hunk 5 above)
Comment (jim) [reply src/scenes/Office/OrdersInfo.jsx]: Nm. I merged master and it is now there.

**Pull request merged**